src/Compiler/LatteToPhpCompiler.php

```diff
@@ -213,6 +217,31 @@ private function addExtractParams(string $phpContent): string
         return $this->printerStandard->prettyPrintFile($phpStmts);
     }
 
+    private function cleanup(string $phpContent): string
+    {
+        $phpStmts = $this->findNodes($phpContent);
+
+        $nodeTraverser = new NodeTraverser();
+        $cleanupVisitor = new class extends NodeVisitorAbstract
```

Comment:
I would create normal class

Reply:
Sure, I will do some refactoring - move all these visitors to some “postprocessor”